site/sass/globals/extends/_cta.scss

```diff
@@ -2,56 +2,53 @@
   $shadow-color: rgba(map-get($google-colors, 'grey-800'), 0.15);
   $this: &;
 
-  @include fontSetup(16px, 24px, 500);
+  @include fontSetup(14px, 16px, 500);
   align-items: center;
   border: 0;
   border-radius: 4px;
   cursor: pointer;
   display: flex;
   font-family: $font-google-sans;
   justify-content: center;
-  padding: 0.75em 1.5em;
+  padding: pxRem(15px) pxRem(40px);
+  min-width: pxRem(150px);
```

Comment:
From my understanding of the designs, CTA's have a greater than 40px left/right spacing when their content results in a < 150px width element. Since the text is centered, setting a min-width allowed for the expansion of the horizontal spacing in this instance without affecting other use cases.